libRALFit/src/ral_nlls_internal.f90

```diff
@@ -693,162 +774,142 @@ recursive subroutine nlls_iterate(n, m, X,                   &
        w%g_old = w%g
     end if
 
-    ! update the stats 
-    inform%obj = 0.5*(w%normF**2)
+    ! update the stats
+    inform%obj = 0.5_wp*(w%normF**2)
     inform%norm_g = w%normJF
     inform%scaled_g = w%normJF/w%normF
+    inform%step = w%normd
     if (options%output_progress_vectors) then
        w%resvec (w%iter + 1) = inform%obj
        w%gradvec(w%iter + 1) = inform%norm_g
     end if
-    
-    if (options%print_level >=1) then
-       write(options%out,1010) w%iter, second, w%Delta, rho, inform%obj, &
-            inform%norm_g, inform%scaled_g
-    end if
+
+!   @TAG:ITERBNR
```

Comment:
tag changed to ITER_BANNER, added comments to describe code segment and output columns